test/libyul/yulSyntaxTests/blobbasefee_reserved_identifier_post_cancun.yul

```diff
@@ -0,0 +1,7 @@
+{
+    function blobbasefee() {}
+}
```

Comment:
```suggestion
{
    {
        let blobbasefee := 1
    }

    {
        function blobbasefee() {}
        blobbasefee()
    }
}
```

Maybe we could keep the same format for the tests across all cancun support prs, i.e. declaring variables and functions with the reserved identifier. For instance, see: https://github.com/ethereum/solidity/pull/14779/files#diff-91342678381e6f1b399e20e7bbcee0c7ee82d170010eb5770950a2977d19b9a7

Reply:
I tried it - it doesn't matter since it will throw the same parser error as it does now, and stop parsing; also, the error is the same regardless of whether it's a function or a variable declaration, which is kinda weird.